docs/reference/api-conventions.asciidoc

```diff
@@ -214,6 +214,37 @@ instead. When using this method, the `source_content_type` parameter
 should also be passed with a media type value that indicates the format
 of the source, such as `application/json`.
 
+[discrete]
+[[api-compatibility]]
+=== REST API version compatibility
+
+Major version upgrades often include a number of breaking changes 
+that impact how you interact with {es}. 
+While we recommend that you monitor the deprecation log and 
```

Comment:
Is there a single deprecation log, or are there multiple log files? Honest question, because I'm not sure 😄

Reply:
Good catch. There are, in fact multiple log files: "By default, Elasticsearch rolls and compresses deprecation logs at 1GB. The default configuration preserves a maximum of five log files: four rolled logs and an active log."

Reply:
```suggestion
While we recommend that you monitor the deprecation logs and 
```